### PR TITLE
media-libs, games-simulation: update SDL, add openttd and related data packages

### PR DESCRIPTION
**Pull request by ElectrodeYT**

### CODE REVIEW

File: bootstrap.d/media-libs.yml [hunk 1]
```diff
@@ -858,8 +858,8 @@ packages:
     source:
       subdir: 'ports'
       git: 'https://github.com/libsdl-org/SDL.git'
-      tag: 'release-2.0.14'
-      version: '2.0.14'
+      tag: 'release-2.28.4'
+      version: '2.28.4'
```
Comment (Dennisbonke) [inline bootstrap.d/media-libs.yml:862]: Please reset revision.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
One comment

File: bootstrap.d/media-libs.yml (hunk 1 above)
Comment (ElectrodeYT) [reply bootstrap.d/media-libs.yml:862]: Done

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**